docs/source/json-api/index.rst

```diff
@@ -819,15 +826,47 @@ HTTP Response
 .. code-block:: json
 
     {
-        "status": 200,
-        "result": [
-            {
-                "party": "Alice",
-                "isLocal": true
-            }
-        ]
+      "status": 200,
+      "result": [
+        {
+          "identifier": "Alice",
+          "displayName": "Alice & Co. LLC",
+          "isLocal": true
+        },
+        {
+          "identifier": "Bob",
+          "displayName": "Bob & Co. LLC",
+          "isLocal": true
+        },
+        {
+          "identifier": "Dave",
+          "isLocal": true
+        }
+      ]
     }
 
+Please note that the order of the party objects in the response is not guaranteed to match the order of the passed party identifiers.
```

Comment:
What happens if there are some known parties and some unknown parties? I expect that we only receive records for the known parties. We should consider adding a `warnings` field containing the unknown parties, but I'm fine doing this in a separate PR.

Reply:
I will add warnings in the follow up PR

Reply:
scratch the above it is a rather small change, easier to add it to this PR and update the docs once.

Reply:
scratch the above it is a rather small change, easier to add it to this PR and update the docs once.